src/app/frontend/secret/list/cardlist.html

```diff
@@ -32,7 +32,7 @@
   </kd-zero-state>
   <kd-resource-card-header-columns ng-show="$ctrl.secretList.listMeta.totalItems">
     <kd-resource-card-header-column size="small"
-                                    grow="4"
+                                    grow="2"
```

Comment:
I run  daborad and  test it,  grow="2"  is better.
![screenshot from 2017-08-16 21-02-46](https://user-images.githubusercontent.com/24387537/29364520-582334f4-8259-11e7-8cb2-f20fd98ef246.png)